### DISCUSSION TITLE
MOOSE/BISON builds but fails to run inputs

### DISCUSSION THREAD

**Discussion by gardnerru** [idaholab/moose#28472] [Q&A Getting Started] [answered]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

I am attempting to build an updated BiSON, while the code builds without error, I get an illegal hardware instruction when running inputs. I tested this with BIsON and the heat_transfer module. I am running Intel mac with OS 14.6.1 and have gone as far as removing moose conda, conda itself, and starting over with the instructions as listed on the BISoN getting started with a new clone but keep hitting the same error. (By new clone I mean rm -rf bison, clone bison, submodule init recursive)
I see that MOOSE still has a testing target for Intel with a slightly older OS that is green, which leads to a user problem I know. A quick google of the error only led to a list of M1 issues. 

Thoughts?

### (Optional) code in question / simulation log / errors

```shell
../../../../heat_transfer-opt -i const_hw.i 
[1]    18209 illegal hardware instruction  ../../../../heat_transfer-opt -i const_hw.i
```

**Comment by milljm** [comment]
This seems to be hitting more and more folks... https://github.com/idaholab/moose/discussions/28340#discussioncomment-10407271

The gist is to downgrade your version of `moose-libmesh` (details in link). This is not a fix, just a workaround until we figure out how to proceed.

**Reply by gardnerru** [reply]
Ahh. I saw that but wasnt sure it would help here. Ill give it a go and report

**Reply by gardnerru** [reply]
@milljm That did work for me. Ill let you decide if you want to mark this as an answer and ill keep watching for updates. Thanks :)

**Reply by milljm** [reply, marked as answer]
We're identifying the issue: https://github.com/idaholab/moose/issues/28484
